Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -603,6 +623,8 @@ def main():
             fetch_incidents()
         elif demisto.command() in ['search', 'es-search']:
             search_command()
+        elif demisto.command() == 'get-mapping-fields':
```

Comment:
Add this command to the yml file too

Reply:
'get-mapping-fields' doesn't need to be declared in the yml. The command is like fetch-incidents - built in and called by the platform, not users. You can double check - Aviad and Slavik will be able to confirm for him.